### ISSUE TITLE
Training speed-up

### ISSUE THREAD

**Issue by dem-esgal** [xuebinqin/U-2-Net#325]
Thank you or your great job!
I believe that you can significantly improve training speed if just increase num_workers in u2net_train:
`salobj_dataloader = DataLoader(salobj_dataset, batch_size=batch_size_train, shuffle=True, num_workers=1)`
